### PR TITLE
Update de_resources.md

### PR DESCRIPTION
**Pull request by wittigue**
- updated list of German RDM guidelines
- added DataPLAN tool

### CODE REVIEW

File: pages/national_resources/de_resources.md [hunk 1]
```diff
@@ -12,6 +12,11 @@ training:
 
 
 national_resources:
+  - name: DataPLAN
+    related_pages:
```
Comment (bedroesb) [inline pages/national_resources/de_resources.md:16]:
```suggestion
    description: fill in
    related_pages:
```

Don't forget to add a description :)

File: pages/national_resources/de_resources.md [hunk 2]
```diff
@@ -12,6 +12,11 @@ training:
...
+      your_tasks: [dmp]
+      your_domain: [plants]
+    url: https://plan.nfdi4plants.org/
   - name: GFBio Data Management Plan Tool
```
Comment (bedroesb) [inline pages/national_resources/de_resources.md:20]: Same here!

**Review by bedroesb — APPROVED** [review]
Looks good on the technical side!

**Comment by bedroesb** [comment]
@lauportell or @Nazeeefa, if you are not on Easter holiday, feel free to review this one too ;)

**Comment by bedroesb** [comment]
I wonder if after this merge, we can close the issue #1222 ?

**Review by Nazeeefa — APPROVED** [review]

**Review by Nazeeefa — APPROVED** [review]

**Review by Nazeeefa — APPROVED** [review]
Looks good! For 'DFG Checklist for handling research data', I put 'PDF document' as the link opens PDF document in browser :)

**Pull request merged**